pkgs/applications/editors/vscode/extensions/vadimcn.vscode-lldb/default.nix

```diff
@@ -126,7 +131,18 @@ in stdenv.mkDerivation {
   '';
 
   passthru = {
-    inherit lldb adapter;
+    inherit lldb;
+    adapter = adapter.overrideAttrs (oa: {
+      postFixup = ''
+        wrapProgram $out/bin/codelldb \
+          --set-default LLDB_DEBUGSERVER_PATH "${lldb.out}/bin/lldb-server" \
+          --add-flags "--liblldb" --add-flags "${lldb.lib}/lib/liblldb.so"
+        # XXX: codelldb expects libcodelldb.so to be in the same
+        # directory as the executable, and can't find it in $out/lib.
+        # This is necessary to make codelldb executable as a standalone.
+        ln -s $out/lib/libcodelldb.so $out/bin/libcodelldb.so
+      '';
```

Comment:
@SuperSandro2000 I've changed this so that only `passthru.adapter` overrides the attrs so that this doesn't interfere with the vscode extension.

Reply:
It was broken for me yesterday on `nixos-unstable` at e92039b55bcd58469325ded85d4f58dd5a4eaf58, but I went to confirm today, and now I can't reproduce the breakage. :confused: It's also working on latest 23.11 and `master`. I had thought your PR was fixing my problem, but I guess I missed whatever actually fixed it.